code/game/area/roguetownareas.dm

```diff
@@ -40,6 +41,13 @@ GLOBAL_LIST_INIT(roguetown_areas_typecache, typecacheof(/area/rogue/indoors/town
 	if((src.warden_area == TRUE) && HAS_TRAIT(guy, TRAIT_WOODSMAN) && !guy.has_status_effect(/datum/status_effect/buff/wardenbuff)) // Warden
 		guy.apply_status_effect(/datum/status_effect/buff/wardenbuff)
 
+
+/area/rogue/Entered(mob/living/carbon/human/guy)
```

Comment:
this is a duplicate definition of `Entered()` (see L51 of this file); it's also horrible object oriented programming-- you could define this on `/area/rogue/indoors/town/church/Entered()` so that you are only running this check if you enter that area, rather than EVERY SINGLE AREA. (obv would have to duplicate it for any church area that doesn't share a path with that but that's still way better than this.)